utilities/ipplan-import.py

```diff
@@ -0,0 +1,181 @@
+"""
```

Comment:
or actually, seeing that you use ipaddress I guess this is py3 only, right? so `python3` :)